src/main/java/Duke.java

```diff
@@ -1,10 +1,99 @@
+                "   |       .-.   .-.       |\n" +
+                "   |      /   \\ /   \\      |\n" +
+                "    \\    | .-. | .-. |    /\n" +
+                "     '-._| | | | | | |_.-'\n" +
+                "         | '-' | '-' |\n" +
+                "          \\___/ \\___/\n" +
+                "       _.-'  /   \\  `-._\n" +
+                "     .' _.--|     |--._ '.\n" +
+                "     ' _...-|     |-..._ '\n" +
+                "            |     |\n" +
+                "            '.___.'\n" +
+                "              | |\n" +
+                "             _| |_\n" +
+                "            /\\( )/\\\n" +
+                "           /  ` '  \\";
+        String greetingMessage = "______________________________________\n" +
+                "Hello! I'm Handsome!\n" +
+                greetingArt +
+                "How can I help you?\n" +
+                "______________________________________\n";
+
+        String goodbyeMessage = "Bye. Hope to see you again soon!\n" + goodbyeArt;
+
+
+        // Flow of chatbot greet -> read input while input != 'bye' -> goodbye
+
+        // Print greeting message
+        System.out.println(greetingMessage);
+
+        // Initialise instance of TaskList
+        TaskList list = new TaskList();
+
+        System.out.println("Please enter your taskList command: (send 'bye' to exit) ");
+        Scanner in = new Scanner(System.in);
+        while (true) {
+            String input = in.nextLine();
+            String[] parsedInput = input.split("\\s+");
+
+            if(parsedInput[0].equals("bye")) {
+                break;
+            } else if (parsedInput[0].equals("list")) {
+                printFormattedMessage(list.getTaskList());
+            } else if (parsedInput[0].equals("mark")){
+                // mark item as complete
+                list.setMarkAsCompleted(Integer.parseInt(parsedInput[1]), true);
+                String markOutput = list.getTaskDescription(Integer.parseInt(parsedInput[1]));
+                printFormattedMessage("Nice! I've marked this task as done:\n" + markOutput);
+            } else if (parsedInput[0].equals("unmark")) {
+                // mark item as incomplete
+                list.setMarkAsCompleted(Integer.parseInt(parsedInput[1]), false);
+                String unmarkOutput = list.getTaskDescription(Integer.parseInt(parsedInput[1]));
+                printFormattedMessage("OK, I've marked this task as not done yet:\n" + unmarkOutput);
+            } else {
+                list.addNewTask(input);
+                printFormattedMessage("added: " + input + "\n");
+            }
+        }
+
+        System.out.println(goodbyeMessage);
+
+    }
+
+    // print formatted message
+    private static void printFormattedMessage(String message) {
+        String divider = "______________________________________\n";
+        System.out.println(divider + message + divider);
```

Comment:
I think this method of separating "divider" first is great, easy for me to understand